modules/data.remote/R/download.thredds.R

```diff
@@ -1,17 +1,27 @@
 #
-##' @title download.thredds.AGB
-##' @name  download.thredds.AGB
+##' @title download.thredds.data
+##' @name  download.thredds.data
```

Comment:
Is `.data` really needed at the end of this name?

Reply:
fixed